source/07-processing/07-federated-searches.rst

```diff
@@ -0,0 +1,23 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Federated Searches
+------------------
+
+The growing use of federated searches and the spread of web crawler robots have the potential to inflate usage statistics, so COUNTER requires you to identify this type of usage in your reports.
+
+Search activity generated by federated search engines MUST be categorized separately from searches conducted by users on the host platform.
+
+Any searches generated from a federated search system MUST be included in the separate Searches_Federated counts within Database Reports and MUST NOT be included in the Searches_Regular or Searches_Automated counts.
+
+The most common ways to recognize federated and automated search activity are as follows:
+
+* A federated search engine may be using its own dedicated IP address, which can be identified and used to separate out the activity.
+* If the standard HTML interface is being used (e.g. for screen scraping), the browser ID within the web log files can be used to identify the activity as coming from a federated search.
+* All searches via APIs and Z39.50 activity (http://www.niso.org/standards/resources/Z39.50_Resources) must be counted as Searches_Federated, as the results are not presented in the platform user interface.
+* If an API gateway is available, set up an instance of the gateway that is for the exclusive use of federated search tools. It is recommended you also require the federated search engine to include an identifying parameter when making requests to the gateway.
```

Comment:
Was it intentional that you've removed the recommendation for Z39.50 but kept the one for the API gateway?